apps/ccd/ithc/base/probatemandb.yaml

```diff
@@ -1,26 +1,27 @@
 apiVersion: v1
 data:
-  DB_PWD: ENC[AES256_GCM,data:zGM7lbHljJxMruAKWKHwzQGCo+E/X1JV,iv:m3ZSfMV9p85qoWViQkGTB24eFYk9x2v75Ngq/Usx2DI=,tag:qCv0jnZFEqCZqnuMnU9Kiw==,type:str]
-  DB_URL: ENC[AES256_GCM,data:gkorDOScyUm0hsjEMI6jw6/BFiV+Cb4DhKGvNsGBlAP4+XmAIfV3zEnG9UthoFvCnu7ouwCKHyPuQvc1Ghz636jY/EKhnEv0ck0JC5+47lWJX7xV2h8yIH/pZPLQow0LRm4zUKdPBUEA7dMSd6vyJakNlEGXUAJrN/qzSj09UBfkJ7L9NHr+H8EtjTuu2Yaj3wqPCYt2/Hc=,iv:dLQ1aLhDIfhztHTXSr6lww6mKoFkA0tDwiIg+Iewto0=,tag:2x8ltg7aJrYwlEeLCwhbrg==,type:str]
-  DB_USER: ENC[AES256_GCM,data:GFlxtN/HSMHQzyJDSagLXC3sxzumWTLDDWbGRXVxamRrNefj9TtgSgRxp2PUM+dj+yxVWt+I/bY1pZKuKd939Woxk5I=,iv:yE3ixzlY2euDSk3oPJq544GJpEOJJ6Y3GTni5NhDIaE=,tag:sI1mFjWVMql38PSwePelOw==,type:str]
+    DB_PWD: ENC[AES256_GCM,data:3DybQo3U0mMKFlaauI6RoNy0WIu1sTgWcVrbUw==,iv:oPtg1Z3ILvVfXn6r1QKVeLmvPFYJoQdAANK74+A6fro=,tag:hUTTDg8a41WpIwJpieRmIw==,type:str]
+    DB_URL: ENC[AES256_GCM,data:ll3nHJ8haPbYISA8rOVT6cQBPWbfkDFPGVURL1uLIUBjdfQwuQ507gp49DjfP0ufbjPuOo26SwvkRpzuSKoN5AIHEnXGKASAIu2Vw3/7Rya9Qmq9zsNfOvIHF3mL+tuZ4F8tEct1vO27NxQQUxjZejetJDFzvJfVOh3UTJ85qPf23Lq0dVELJ7YSEDf5ifLHt+rRJ4be6E/IAzwWOZTVmg==,iv:F6ai671Y4wTufhpY3zYAEpjPtgvr6nb881Ny5TReY64=,tag:RBeDoLgRZFK2OATEoEQVyw==,type:str]
+    DB_USER: ENC[AES256_GCM,data:gHSTvYCx5xc6557A,iv:us7+QoLhGobspxMAxdJHWG2w0NB9I7YAb3CFpDSvTGE=,tag:+Y81H2N7HCjybWUBdyEgPg==,type:str]
 kind: Secret
 metadata:
-  name: probatemandb
-  namespace: ccd
+    creationTimestamp: null
+    name: probate
```

Comment:
Why are we changing the name of the secret from `probatemandb`  to `probate` ? this will require further change wherever that secret has been used

Reply:
oversight on my part when writing the kubectl commands. This was also done on AAT so i'll update to fix

Reply:
This should be resolved now